modules/k6/src/main/java/org/testcontainers/k6/K6Container.java

```diff
@@ -0,0 +1,98 @@
+package org.testcontainers.k6;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class K6Container extends GenericContainer<K6Container> {
+
+    /** Standard image for k6, as provided by Grafana. */
+    public static final DockerImageName K6_IMAGE = DockerImageName.parse("grafana/k6:0.49.0");
+
+    /**
+     * Extended image allowing for dynamic inclusion of k6 Extensions.
+     * @see <a href="https://grafana.com/docs/k6/latest/extensions/">k6 Extensions</a>
+     */
+    public static final DockerImageName K6_BUILDER_IMAGE = DockerImageName.parse("szkiba/k6x:v0.4.0");
+
```

Comment:
I'll go with the default one. Users can mark this image as a valid substitute in they want to use it.

```suggestion
```